### PR TITLE
Add .minerva v3.5.0

### PR DESCRIPTION
**Pull request by Zirro**
Add .minerva, a flash shared object editor. The application has been
discontinued in favour of a version hosted online, but remains
available on the homepage and is desirable for offline use. A caveat
has been added to reflect this.

### CODE REVIEW

File: Casks/minerva.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask :v1 => 'minerva' do
+  version '3.5.0'
+  sha256 'c2a53ab7bd14924e2a7e3f987eae2586f408c6b24b8b59cad435f9bd7668a130'
+
+  url "http://blog.coursevector.com/downloads/Minerva-#{version.gsub('.', '-')}.air"
+  name '.minerva'
+  homepage 'http://blog.coursevector.com/minerva/'
+  license :gratis
+
+  depends_on :cask => 'adobe-air'
```
Comment (adityadalal924) [inline Casks/minerva.rb]: Please  add an empty line after this one.

**Comment by adityadalal924** [comment]
Can you confirm that this Cask works on the latest OS X? (else another depends_on needs to be added)

**Comment by Zirro** [comment]
Yes, it runs on El Capitan.

Unfortunately, it looks like my attempt at squashing commits may have caused some collateral damage. If this can not be resolved on your end, I will start over with a new pull request tomorrow.

**Comment by adityadalal924** [comment]
Here's how to fix this, assuming your repository is set up like (you can get this info with `git remote -v`):
```
adityadalal924	https://github.com/adityadalal924/homebrew-cask (fetch)
adityadalal924	https://github.com/adityadalal924/homebrew-cask (push)
origin	https://github.com/caskroom/homebrew-cask (fetch)
origin	https://github.com/caskroom/homebrew-cask (push)
```
You can use `git rebase -i origin/master` (or whatever you called the upstream repo) and use f for all commits after the first one. Then do a `git push "adityadalal924" branch_name" --force` (replace adityadalal924 with your repo's name).

**Pull request closed without merging**